### ISSUE TITLE
New diagnostic warnings will be confusing until default output is changed

### ISSUE THREAD

**Issue by jgabry** [stan-dev/rstan#623]
@avehtari and I are talking about this at the moment and we think it will be very confusing to users to warn them about the new MCMC diagnostics (e.g, bulk and tail ess)

https://github.com/stan-dev/rstan/blob/e0704f1f4158a26a0a831a85a179a694cd17182d/rstan/rstan/R/check_hmc_diagnostics.R#L86

when these new diagnostics are not yet included in the default output. If we aren't going to wait for the new diagnostics to be the default output before warning users about them then I think we should at least point them in the warning message to the functions that can be used to access these quantities instead of just giving the URL to the Stan warnings document.

**Comment by bgoodri** [comment]
I don't think regular users should be calling things like tail_ess(). We
could put in the message that they should call monitor() but that is only
because we can't change the default output without people complaining about
things that are much less important that chains that do not converge or mix
well. But if we don't throw warnings, then no one know to call monitor().

**Comment by jgabry** [comment]
I agree with all that. But how about we suggest calling monitor_new or whatever it’s called in the warning message? That wouldn’t change the default output but would point users to the new output.

**Comment by bgoodri** [comment]
We could change the message from saying things like "tail_ess was too low
..." to "tail_ess was 162, which ...". I don't think we should be creating
an alternative workflow where users are having to call the good version of
summary in order to see what they need to see because we are incapable of
putting that into the existing version of summary.

**Comment by jgabry** [comment]
ok I think the warnings should at least mention calling monitor to see the new diagnostics. Maybe we can just add “Call monitor(stanfit) for additional diagnostics” to the end of the new warnings (before the recommended url) until they are included in the default output someday?

And I agree that the reason we are waiting to change summary is less important than knowing if the chains did well, but seems like we’re stuck with that? I don’t see why we can’t leave “mean” as the first column of the internal structure but change the first thing printed in the output, but I guess that’s been ruled out.
